### ISSUE TITLE
Polar playground

### ISSUE THREAD

**Issue by danmx** [osohq/oso#1049]
Something similar to [Rego Playground](https://play.openpolicyagent.org/) would be awesome

**Comment by gneray** [comment]
Yes, we completely agree! This is on the roadmap, just a matter of timing. 

Out of curiosity: what would be the main things you'd want to try / achieve with a playground? Or said differently, what are the main things that are currently hard with Polar, and that having a playground could help with?

**Comment by danmx** [comment]
I'm trying to compare Polar to Rego. I have some experience with OPA but I find Rego learning curve quite steep, syntax not very intuitive and integrations with other languages than Go are limited so I was looking for alternatives.
